PokeAlarm/Filters/EggFilter.py

```diff
@@ -46,6 +47,18 @@ def __init__(self, name, data):
             limit=BaseFilter.parse_as_set(
                 GymUtils.create_regex, 'gym_name_contains', data))
 
+        # Gym sponsor
```

Comment:
You need to add new unit tests when adding a new restriction to a filter.

Reply:
Can you elaborate?

Reply:
Look in `tests/filters/test_egg_filter.py` and `tests/filters/test_raid_filter.py`, these files create the unit testing for all Raid & Egg Filters.  This is what @kvangent is wanting you to create, add in a unit test to test that your filters work.  

Once you have created the unit test, you can test it by running `python -m unittest discover`